src/applications/static-pages/facilities/BasicFacilityListWidget.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { apiRequest } from '../../../platform/utilities/api';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import { buildAddressArray } from '../../facility-locator/utils/facilityAddress';
+
+export default class BasicFacilityListWidget extends React.Component {
```

Comment:
What is the difference between this widget an the other facility list one? We might want to figure out a way to reuse some of the duplicate code between them.

Reply:
It's a subset of that one. It doesn't have the image or button link. Where would be a good place to put the shared code?

Reply:
I think in the same folder here.